---
title: ABLeS available software and reference data
toc: false
---

This page provides a comprehensive overview of tools, containers, workflows, and datasets available to ABLeS researchers. It also includes information about the **Australian BioCommons Tools and Workflows project (if89)**, which serves as a shared repository for bioinformatics tools and workflows.

### Pawsey Supercomputing Centre

#### Software

There is a wide range of centrally installed software for Setonix users, including popular bioinformatics tools. More information about [software on Setonix](https://support.pawsey.org.au/documentation/display/US/Software+Stack).

Setonix users can also install their choice of software on the `/software` partition. We also encourage usage of containers with Singularity. Users are able to download any containers they like!

#### Reference datasets

Popular bioinformatics [reference datasets](https://support.pawsey.org.au/documentation/display/US/Life+Science+and+Bioinformatics) are available on Pawsey. Additional reference sets can be requested via <help@pawsey.org.au>

### National Computational Infrastructure (NCI)

#### Centrally supported software (`/apps`)

Centrally supported software available through NCI can be viewed [here](https://opus.nci.org.au/display/Help/5.+Software+Applications).

### Shared repository of tools and software (`project if89` on NCI)

Users at NCI also have access to the `Australian BioCommons Tools and Workflows project`, in project allocation `if89`. This is a repository of popular tools, containers and workflows that can be used by anyone in the NCI system.

Anyone from an NCI project is also invited to contribute to `if89`, and add more software installations that can be shared with others.

#### Software

The list of tools available through the Australian BioCommons Shared Tools and Workflows repository (`NCI (if89)`) is available through <a href="https://australianbiocommons.github.io/2_tools.html">ToolFinder</a>.

 <div class="accordion" id="accordion-if89-tool">
      <!--<div class="accordion-item">
        <h2 class="accordion-header" id="heading-if89-tool-1" style="margin-top:0rem">
          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-if89-tool-1" aria-expanded="false" aria-controls="collapse-if89-tool-1">
            List of all available tools in if89  (2022-04-13)
          </button>
        </h2>
        <div id="collapse-if89-tool-1" class="accordion-collapse collapse" aria-labelledby="heading-if89-tool-1" data-bs-parent="#accordion-if89-tool">
          <div class="accordion-body" >
            The frequently updated list of tools available through the Australian BioCommons Shared Tools and Workflows repository is available though Tool Finder webpage (<a href="https://australianbiocommons.github.io/2_tools.html">Project if89 on GADI-NCI</a> )
          </div>
        </div>
      </div> -->
      <div class="accordion-item">
        <h2 class="accordion-header" id="heading-if89-tool-2" style="margin-top:0rem">
          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-if89-tool-2" aria-expanded="false" aria-controls="collapse-if89-tool-2">
            Utilising if89 tools
          </button>
        </h2>
        <div id="collapse-if89-tool-2" class="accordion-collapse collapse" aria-labelledby="heading-if89-tool-2" data-bs-parent="#accordion-if89-tool">
          <div class="accordion-body">
            In order to load any modules from if89, you need to prepend the directory of if89's modules to the MODULEPATH environment variable using the following command: <br/>
            <code>module use -a /g/data/if89/apps/modulefiles</code> <br/>
            After that, you can load any tool, then utilising it directly using the following command:<br/>
            <code>module load $tool/$version</code> <br/>
            You can list all available modules using the following command:<br/>
            <code>module available</code>
          </div>
        </div>
      </div>
        <div class="accordion-item">
                <h2 class="accordion-header" id="heading-if89-tool-3" style="margin-top:0rem">
                  <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-if89-tool-3" aria-expanded="false" aria-controls="collapse-if89-tool-3">
                    Contributing to if89 repository
                  </button>
                </h2>
                <div id="collapse-if89-tool-3" class="accordion-collapse collapse" aria-labelledby="heading-if89-tool-3" data-bs-parent="#accordion-if89-tool">
                  <div class="accordion-body">
                   Please see <a href="/ables/if89-technical/">documentation</a> for more information.
                  </div>
                </div>
              </div>
 </div>

<br/>

#### Containers

 <div class="accordion" id="accordion-if89-cont">
      <div class="accordion-item">
        <h2 class="accordion-header" id="heading-if89-cont-1" style="margin-top:0rem">
          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-if89-cont-1" aria-expanded="false" aria-controls="collapse-if89-cont-1">
            List of all available Containers
          </button>
        </h2>
        <div id="collapse-if89-cont-1" class="accordion-collapse collapse" aria-labelledby="heading-if89-cont-1" data-bs-parent="#accordion-if89-cont">
          <div class="accordion-body" >
            Will be available soon.
        </div>
        </div>
      </div>
 </div>

<br/>

#### Workflows

 <div class="accordion" id="accordion-if89-wf">
      <div class="accordion-item">
        <h2 class="accordion-header" id="heading-if89-wf-1" style="margin-top:0rem">
          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-if89-wf-1" aria-expanded="false" aria-controls="collapse-if89-wf-1">
            List of all available Workflows
          </button>
        </h2>
        <div id="collapse-if89-wf-1" class="accordion-collapse collapse" aria-labelledby="heading-if89-wf-1" data-bs-parent="#accordion-if89-wf">
          <div class="accordion-body" >
            Will be available soon.
        </div>
        </div>
      </div>
 </div>

<br/>

#### Software databases

Some of the databases required by different bioinformatics software tools are made available through the if89 project.  
They are located at <code>/g/data/if89/data_library</code>. You can request other databases to be included by [contacting us](https://australianbiocommons.github.io/ables/contact-us/).

A list of the currently available (`as of 25 Jan 2023`) databases is included below:

<div class="accordion" id="accordion-if89-ds">
      <div class="accordion-item">
        <h2 class="accordion-header" id="heading-if89-ds-1" style="margin-top:0rem">
          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-if89-ds-1" aria-expanded="false" aria-controls="collapse-if89-ds-1">
            List of all available databases
          </button>
        </h2>
        <div id="collapse-if89-ds-1" class="accordion-collapse collapse" aria-labelledby="heading-if89-ds-1" data-bs-parent="#accordion-if89-ds">
          <div class="accordion-body" >
            <table>
              <thead>
                <tr>
                  <th><strong>Dataset</strong></th>
                  <th><strong>Source</strong></th>
                  <th><strong>Download date</strong></th>
                  <th><strong>Location</strong></th>
                  <th><strong>Details</strong></th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>Blast</td>
                  <td><a href="https://blast.ncbi.nlm.nih.gov/doc/blast-help/downloadblastdata.html"> Blast Webpage </a></td>
                  <td> 28 Aug 2022 </td>
                  <td><code>blast_db/28082022/</code></td>
                  <td> <strong>nr.*.gz</strong>: non-redundant protein sequence database with entries
                           from GenPept, Swissprot, PIR, PDF, PDB, and RefSeq. <strong>nt.*.gz</strong>: nucleotide sequence database, with entries from all
                          traditional divisions of GenBank, EMBL, and DDBJ.
                  </td>
                </tr>
                <tr>
                   <td>Blast</td>
                   <td><a href="https://blast.ncbi.nlm.nih.gov/doc/blast-help/downloadblastdata.html"> Blast Webpage </a></td>
                   <td> 7 Nov 2023 </td>
                   <td><code>blast_db/07112023</code></td>
                   <td><strong>nt.*</strong>: nucleotide sequence database, with entries from all
                     traditional divisions of GenBank, EMBL, and DDBJ.
                   </td>
                </tr>
                <tr>
                  <td>Alphafold/UniProt</td>
                  <td> <a href="https://github.com/steineggerlab/foldseek"> foldseek github pages</a> and <a href="https://alphafold.ebi.ac.uk/"> AlphaFold Protein Structure Database webpage </a></td>
                  <td>30 Nov 2022</td>
                  <td><code>AlphaFoldDB/aminoacid/UniProt/30112022/</code></td>
                  <td>Aminoacid dataset for foldseek tool. Downloaded through <code>databases</code> command in foldseek tool.</td>
                </tr>
                <tr>
                  <td> Alphafold/UniProt-NO-CA</td>
                  <td> <a href="https://github.com/steineggerlab/foldseek"> foldseek github pages</a> and <a href="https://alphafold.ebi.ac.uk/"> AlphaFold Protein Structure Database webpage </a></td>
                  <td>30 Nov 2022</td>
                  <td><code>AlphaFoldDB/aminoacid/UniProt-NO-CA/30112022/</code></td>
                  <td>Aminoacid dataset for foldseek tool. Downloaded through <code>databases</code> command in foldseek tool.</td>
                </tr>
                <tr>
                  <td>Alphafold/UniProt50</td>
                  <td> <a href="https://github.com/steineggerlab/foldseek"> foldseek github pages</a> and <a href="https://alphafold.ebi.ac.uk/"> AlphaFold Protein Structure Database webpage </a></td>
                  <td>30 Nov 2022</td>
                  <td><code>AlphaFoldDB/aminoacid/UniProt50/30112022/</code></td>
                  <td>Aminoacid dataset for foldseek tool. Downloaded through <code>databases</code> command in foldseek tool.</td>
                </tr>
                <tr>
                  <td>Alphafold/Proteome</td>
                  <td> <a href="https://github.com/steineggerlab/foldseek"> foldseek github pages</a> and <a href="https://alphafold.ebi.ac.uk/"> AlphaFold Protein Structure Database webpage </a></td>
                  <td>29 Nov 2022</td>
                  <td><code>AlphaFoldDB/aminoacid/Proteome/29112022/</code></td>
                  <td>Aminoacid dataset for foldseek tool. Downloaded through <code>databases</code> command in foldseek tool.</td>
                </tr>
                <tr>
                  <td>Alphafold/Swiss-Prot</td>
                  <td> <a href="https://github.com/steineggerlab/foldseek"> foldseek github pages</a> and <a href="https://alphafold.ebi.ac.uk/"> AlphaFold Protein Structure Database webpage </a></td>
                  <td>30 Nov 2022</td>
                  <td><code>AlphaFoldDB/aminoacid/Swiss-Prot/30112022/</code></td>
                  <td>Aminoacid dataset for foldseek tool. Downloaded through <code>databases</code> command in foldseek tool.</td>
                </tr>
                <tr>
                  <td>Busco/eukaryota_odb10</td>
                  <td> <a href="https://busco-data.ezlab.org/v5/data/lineages/"> Busco webpages</a></td>
                  <td>14 Aug 2023</td>
                  <td><code>busco_db/14082023/lineages</code></td>
                  <td>Lineage datasets for busco tool. Downloaded manually.</td>
                </tr>
                <tr>
                 <td>Kaiju</td>
                 <td><a href="https://bioinformatics-centre.github.io/kaiju/downloads.html"> Kaiju Webpage </a></td>
                 <td> 26 May 2023 </td>
                 <td><code>kaiju/26052023/kaiju_db_rvdb</code></td>
                 <td>Kaiju pre-built indexes for protein sequences from RVDB-prot v26.0. Contains the Kaiju .fmi index file, as well as nodes.dmp and names.dmp from the NCBI taxonomy.
                 </td>
               </tr>
               <tr>
                 <td>Kraken2</td>
                 <td><a href="https://benlangmead.github.io/aws-indexes/k2"> Kraken 2, KrakenUniq and Bracken indexes </a></td>
                 <td> 9 Oct 2023 </td>
                 <td><code>kraken2/09102023/k2_pluspf</code></td>
                 <td>Kraken2 pre-built index for RefSeq database (archaea, bacteria, viral, plasmid, human, protozoa & fungi) plus UniVec_Core.
                 </td>
               </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
  </div>

  <br/>
  <br/>

## if89 Contributors

{% include contributor-tiles-all.html custom="Hardip Patel, Ziad Al Bkhetan, J King Chang, Andre Martins Reis, Hasindu Gamaarachchi, Kyle Drover, Tim Amos, Kisaru Liyanage, Terry Bertozzi, Javed Shaikh, Kirat Alreja, Leah Kemp, Andrey Bliznyuk, Hyungtaek Jung, Wenjing Xue, Johan Gustafsson, Dale Roberts" sort=false%}
