---
title: Quantitative Proteomics Analysis of Drought Tolerance in Australian Wild Cotton Species
description: This project investigates drought tolerance in Australian wild cotton species through quantitative proteomics analysis, comparing protein expression under drought and normal conditions. We will annotate protein sequences for three species using publicly available genomic and transcriptomic data, aiding our understanding of their drought response mechanisms.
toc: false
type: ABLeS Participant
---

## Project title

Quantitative Proteomics Analysis of Drought Tolerance in Australian Wild Cotton Species

## Collaborators and funding

The project involves collaboration between the Australian Proteome Facility and The School of Natural Sciences at Macquarie University.
The project was funded by Bioplatforms Australia.

## Contact(s)

- Ignatius Pang, Australian Proteome Analysis Facility (APAF), <ignatius.pang@mq.edu.au>
- Brian Atwell, School of Natural Sciences, Macquarie University, <brian.atwell@mq.edu.au> 

## Project description and aims

The aim of the project is to perform quantitative proteomics analysis on five species of cotton in drought versus control conditions, to study drought tolerance of Australian wild cotton species. To perform searches of the mass spectrometry data, we have protein sequence database for two species (G. australe and G. hirsutum). However, while genomic data is available, the protein sequence database were not yet annotated for three species (G. bickii, G. sturtianum, and G. robinsonii). This project involves using publicly available data to perform protein sequence annotation on the three genomes. The genome of G. robinsonnii will be assembled using short-reads sequencing data available on NCBI RSA database (PRJNA592601). Short-reads RNA-seq data for G. bickii will be downloaded from NCBI (PRJNA592601) and assembled using Trinity to perform de novo transcriptome assembly. The assembled transcriptome will be used to assist with G. bickii protein annotations using the Maker tool. The G. bickii transcriptome sequences will be used to aid protein annotations for G. sturtianum and G. robinsonnii using the Maker tool as Maker allows the use of transcripts from closely related species to assist with protein annotations. The quantitative proteomics analysis will be performed using the ProteomeScholaR pipeline developed by APAF: [https://github.com/APAF-bioinformatics/ProteomeScholaR](https://github.com/APAF-bioinformatics/ProteomeScholaR).

## How is ABLeS supporting this work?

This work is supported through the production bioinformatics scheme provided by ABLeS. The supports includes unlimited temporary storage on scratch, 3 TB permanent storage and 100 KSUs allocation per quarter.

## Expected outputs enabled by participation in ABLeS

The outcome research will be written as a manuscript and submitted to a peer reviewed journal for publication. The assembled genome and protein annotations will be submitted to public genomics repository (e.g. NCBIâ€™s databases). 

<br/>

> _These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above._
