---
title: Workspace for Genomics for Australian Plants (GAP) Framework Initiative
description: Development of genomics resources to enhance our understanding of the evolution and conservation of the unique Australian flora.
toc: false
type: ABLeS Participant
---


## Collaborators and funding

The Genomics for Australian Plants Framework Initiative was initiated by Bioplatforms in partnership
with researchers from the Australian State and National Herbaria and Botanic Gardens.

The Initiative is supported by funding from Bioplatforms Australia (enabled by NCRIS), the Ian Potter Foundation, Royal Botanic Gardens Foundation (Victoria), Royal Botanic Gardens Victoria, the Royal Botanic Gardens and Domain Trust, the Council of Heads of Australasian Herbaria, CSIRO, Centre for
Australian National Biodiversity Research and the Department of Biodiversity, Conservation and
Attractions, Western Australia.

## Contact(s)

- GAP Bioinformatics Lead: Dr Theodore Allnutt <Theodore.Allnutt@rbg.vic.gov.au>
- GAP Phylogenomics Bioinformatics Lead: Dr Alexander Schmidt-Lebuhn <Alexander.S-L@csiro.au>
- GAP Project Lead: Prof David Cantrill <David.Cantrill@rbg.vic.gov.au>
- GAP Project Manager: Dr Mabel Lum <mlum@bioplatforms.com>

## Project description and aims


[Genomics for Australian Plants](https://www.genomicsforaustralianplants.com/) was initiated by Bioplatforms in partnership with researchers from the Australian State and National Herbaria and Botanic Gardens.

Broadly, the key aims of the Initiative are:

+ Build genomic capacity across Australian Botanic Gardens and Herbaria to create networks collaborating in the collection, management, dissemination and
application of genomic data for Australian plants;

+ Provide tools to enable genomic data to be used to identify and classify biodiversity
at a range of scales and to use these tools to inform conservation management and
enable better decision making.

## How is ABLeS supporting this work?

This work is supported through the reference data asset creation scheme provided by ABLeS. The support includes 33 TB long term storage, 1 TB temoprary storage on scratch and 400 KSUs per quarter.

## Expected outputs enabled by participation in ABLeS

1. Approximately 2000 samples&#39; phylogenomic assemblies of Angio-353 genes. The
assemblies, and alignments and trees generated from them will be used to publish the
Australian Angiosperm Tree of Life (AAToL) and will contribute data to the Plants and Fungi Tree of Life (PAFToL - led by Kew Gardens, UK).

2. Within-species phylogenomic assemblies of Angio-353 for the genera in the GAP Stage 2 phylogenomics project.

<br/>

> *These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above.*
