---
title: Federation University Australia
description: Molecular Dynamics Studies of Horse Prion Protein S167D-Variant and Wild-Type, and Other PrPC Proteins and PrPScs.
toc: false
type: ABLeS Participant
---

## Project title

Molecular Dynamics Studies of Horse Prion Protein S167D-Variant and Wild-Type

## Collaborators and funding

[Federation University Australia](https://www.federation.edu.au/)

## Contact(s)

-   Jiapu Zhang, Federation University Australia, <j.zhang@federation.edu.au>

## Project description and aims

Prion diseases, also called transmissible spongiform encephalopathies (TSEs), are fatal neurodegenerative diseases characterised by the accumulation of an abnormal prion protein isoform (PrPSc: rich in β-sheets—about 30% α-helix and 43% β-sheet), which is converted from the normal prion protein (PrPC: predominantly α-helical—about 42% α-helix and 3% β-sheet). However, prion disease has not been reported in horses up to now; therefore, horses are known to be a species resistant to prion diseases. Residue S167 in horses has been cited as a critical protective residue for encoding PrP conformational stability in prion-resistance. According to the “protein-only” hypothesis, PrPSc is responsible for both the spongiform degeneration of the brain and disease transmissibility. Thus, understanding the conformational dynamics of PrPSc from PrPC is key to developing effective therapies. This project focuses on molecular dynamics (MD) studies on the horse PrP S167D mutant (S167D-Variant) and Wild Type (WT), in order to understand their conformational dynamics. My WT MD results in this article (Zoonotic Dis. 2024, 4(3):187-200) have confirmed that the single amino acid differences at position 167 might influence the overall protein structures of the WT.

In my article "Zhang J (2024) Molecular Dynamics and Optimization Studies of Horse Prion Protein Wild Type and Its S167D Mutant. Zoonotic Dis. 2024, 4(3):187-200; https://doi.org/10.3390/zoonoticdis4030017" (denoted as "the Article" in this small project), the investigator has not finished many aspects of the MD simulations:

(i) the S167D-Variant should be done, because as it is reported by the Article "this article reports no results from the MD simulations for the mutant", "Various secondary structural statistics for the mutant simulations at different temperatures and pH conditions, and the changes in the H-bond network caused by the mutation with MD simulations, should be presented";

(ii) the WT should be done MD simulations furthermore from the 30 ns; because as it is reported by the Article  "MD simulations on a timescale (30 ns) much shorter than the one we required to observe significant protein conformational change; it is therefore still unclear how probative these simulations are of the biological process we are trying our best to model", "a longer MD timescale is still needed to understand further. The simulations are very short (30 ns) to verify structural change. The simulation time should be increased and the conformational changes should be noted as soon as the computing resources are available from NCI Australia";

(iii) the MD should be done for the interactions of WT with the solvent and the ions, ligands binding, as it is reported in the Article "the free energy calculations are a research direction for this article to investigate the effects/contributions of ions such as Cu2+ and of solvents such as water. This should be highlighted as a future research direction for the author".

**Project background:**

Prion diseases are incurable neurodegenerative diseases caused by aberrant conformations of the prion protein (PrP). Many animals develop similar diseases, but rabbits, dogs, and horses show unusual resistance to prion diseases This resistance could be due to protective changes in the sequence of the corresponding PrP in each animal. Structural studies have identified S174, S167 and D159 as the key protective residues in rabbit, horse and dog PrP, respectively. But no systemic MD studies currently support the protective activity of these residues, especially for the horse PrP residue S167. Experimental laboratory results revealed that expression of horse PrP-S167D (which carries a substitution for the equivalent residue in the PrP of hamsters, a species that is susceptible to prion diseases) shows high toxicity in behavioural and anatomical assays. Thus, this project aims to carry out an MD study of the horse PrP wild-type (WT) NMR structure 2KU4.pdb and the S167D mutant (hereafter, mutant) homology structure (constructed by me). We will present in this project useful protective bioinformatics of S167 and discuss the structural features that make horse PrP more stable. The findings of this project might contribute to the development of drugs/compounds that stabilize the PrP structure and prevent the formation of toxic conformations of prion diseases.

Here, we detail the central topic on PrP more in this brief introduction. Unlike bacteria and viruses which are based on DNA and RNA, prions are unique as disease-causing agents since they are misfolded proteins. Prion contains no nucleic acids, and it is a misshapen or conformationally changed protein that acts like an infectious agent. Prion diseases are called “protein structural conformational” diseases. Normal prion protein is denoted as PrPC and diseased infectious prion is denoted as PrPSc. PrPC is predominant in α-helices, but PrPSc is rich in β-sheets in the form of amyloid fibrils. PrPC is a normal protein found on the membranes of cell, including several blood components of which platelets constitute the largest reservoir in humans. Several topological forms exist; one cell-surface form anchored via glycolipid and two transmembrane forms. The normal protein has a complex function, which continues to be investigated at present; the cleavage of PrPC in peripheral nerves causes the activation of myelin repair in Schwann cells, PrPC regulates cell death, PrPC may have a function in the maintenance of long-term memory, PrPC may play roles in innate immunity and stem cell renewal, etc. PrPC binds Cu2+ ions with high affinity; the significance of this property is not clear, but it is presumed to relate to the protein’s structure or function. PrPC is not sedimentable, meaning it cannot be separated by centrifuging techniques. PrPC is readily digested by proteinase K and can be liberated from the cell surface by the enzyme phosphainositide phospholipase C, which cleaves the glycophosphatidylinositol glycoplipid anchor. PrPC plays an important role in cell–cell adhesion and intracellular signaling in vivo and may therefore be involved in cell–cell communication in the brain. PrPSc always causes prion disease. Several highly infectious, brain-derived PrPSc structures have been discovered by cryo-EM; another brain-derived fibril structure isolated from humans with the prion disease GSS syndrome has also been determined. Often, PrPSc is bound to cellular membranes, presumably via its array of glycolipid anchors; however, sometimes the fibres are dissociated from membranes and accumulate outside of cells in the form of plaques. S167 in PrPC is a protective residue and generates a more compact and stable structure in the C-terminal subdomain of the PrPC global domain.

## How is ABLeS supporting this work?



## Expected outputs enabled by participation in ABLeS

The research results will be published by a Springer Nature monograph or several journals' articles.

<br/>

> _These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above._
