---
title: NACL lab, University of Newcastle
description: Salt extremophiles (plant halophytes) have unique salt tolerance mechanisms not found in food crops. New genomic, transcriptomic, proteomic and genetic resources are now unlocking genes and pathways that could be introduced to our food crops. At the same time, we are using these resources to guide our modern breeding leading to domesticated wild extremophiles as future food crops.
toc: false
type: ABLeS Participant
---

## Project title

Salt extremophiles and future food crops

## Collaborators and funding

[University of Newcastle](https://www.newcastle.edu.au/)

## Contact(s)

Vanessa Melino, University of Newcastle, <vanessa.melino@newcastle.edu.au>

## Project description and aims

- **Aim 1.** Define and resolve the phylogenetic relationships of novel class of RNA-binding proteins across the plant kingdom and marine photosynthetic organisms.
*Planned Analysis 1.* Collecting >2000 protein sequences across organisms, multiple sequence alignments and maximum likelihood calculations.
- **Aim 2.** Identify RNA-binding targets and common binding motifs for this novel class of RNA-binding proteins from salt extremophile when expressed in GM Arabidopsis and exposed to salt stress.
*Planned Analysis 2.* RNA-immunoprecipitation to isolate and sequence RNA binding partners by RIP-seq and input RNA by RNA-seq. Quality control and adapter trimming of raw reads, alignment to reference, read counts and enrichment analysis. Differential expression analysis and GO ontology and functional annotation.
- **Aim 3.** Identify new gene targets and regulatory elements for unique salt tolerance mechanisms identified in wild extremophiles (Salicornia genus).
*Planned Analysis 3.* Visualisation of RNA-seq and IsoSeq data on new reference genomes for six Salicornia species, improved annotation using Apollo version 3.
- **Aim 4.** Genome-wide association study of domestication traits across a genotyped population of a wild extremophile (Salicornia genus).
*Planned Analysis 4.* Seed quality traits with equal distribution across the diploid Salicornia population used in GWAS testing for differences in allele frequency with different models. Bioinformatic visualization of SNPs across population as compared to the reference genome.

**Outcomes:** This project will generate new knowledge on a unique salt tolerance mechanism enabling salt extremophiles to grow with salt water. Regulatory pathways will be identified bioinformatically and tested experimentally in a synthetic biology approach. ABLeS will also enable our team to use our extensive omic resources for better understanding of hpw the pathways are regulated forming the basis of a new sub-discipline in stress adaptation research.
[https://www.newcastle.edu.au/profile/vanessa-melino](https://www.newcastle.edu.au/profile/vanessa-melino)
Research Aims 1 and 2. submitted for ARC DP26 and awaiting review.

## How is ABLeS supporting this work?

Pawsey. Access to CPU mostly but at specific times to GPUs. Storage required for project duration. 3 expected users for 2025. Project duration 2025-2028.

## Expected outputs enabled by participation in ABLeS

Outputs published in peer-reviewed journals. Data published on NCBI and/or Dryad database. Bioinformatic workflows published in GitHub.

<br/>

> _These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above._
