---
title: Northcott Neuroscience Laboratory at ANZAC Research Institute
description: Development of a workflow for Structural variation detection and filtration.
toc: false
type: ABLeS Participant
---

## Project title

Whole genome structural variant profiling in heritable neuropathies

## Collaborators and funding

- Northcott Neuroscience Laboratory, ANZAC Research Institute, University of Sydney 

- Sydney Informatics Hub, University of Sydney 

Funded via a MRFF Genomics Health Futures Mission grant (APP2007681) 


## Contact(s)

- Georgie Samaha, Sydney Informatics Hub, University of Sydney <georgina.samaha@sydney.edu.au>

- Marina Kennerson, ANZAC Research Institute, University of Sydney <marina.kennerson@sydney.edu.au>



## Project description and aims

For many inherited disorders of the motor neurons such as hereditary motor neuropathy (a form of inherited peripheral neuropathy; IPN) and amyotrophic lateral sclerosis (ALS), there has been success in identifying causative genes using whole exome sequencing. 
Despite this success, up to 40% of familial cases remain genetically unsolved. Structural variation including balanced (insertions, inversion, translocation), unbalanced duplication/deletions (copy number variation; CNV) and repeat expansions is an important class of mutation which has been understudied in these disorders. 

We aim to develop a pipeline that will detect SV across the genome and filter variants identified by selecting those located close to known causative genes. This project will foster an important paradigm shift from exome analysis to rigorously investigating the remaining 98% of the genome using WGS to identify mutations. 

## How is ABLeS supporting this work?

This work is supported through the production bioinformatics scheme provided by ABLeS. The support includes 2 TB long term storage, 11 TB temoprary storage on scratch and 50 KSUs per quarter.

## Expected outputs enabled by participation in ABLeS
Our research outcomes will help spearhead the motor neuron disease field into the fast-approaching future of WGS diagnostic screening, in which utilising tools for detection and clinically interpreting SVs will be essential to maximising the success of genetic diagnosis in genetically unsolved families.

A computational workflow for structural variant detection and functional annotation in whole genome sequence data. The workflow development can be followed on [GitHub](https://github.com/Sydney-Informatics-Hub/Germline-StructuralV-nf). 

<br/>

> *These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above.*
