---
title: Queensland University of Technology (QUT)
description: ONTViSc is a Nextflow-based end-to-end bioinformatics pipeline designed to help diagnostics of viruses and viroid pathogens for biosecurity.
toc: false
type: ABLeS Participant
---

## Project title

Development of an ONTViSc (ONT-based Viral Screening for Biosecurity) pipeline.

## Collaborators and funding

- Department of Agriculture, Fisheries and Forestry https://www.agriculture.gov.au/

- Australian BioCommons https://www.biocommons.org.au/

- Queensland University of Technology https://www.qut.edu.au/

- Queensland Cyber Infrastructure Foundation https://www.qcif.edu.au/

## Contact(s)

Queensland University of Technology (QUT)

- Marie-Emilie Gauthier gauthiem@qut.edu.au

- Roberto Barrero roberto.barrero@qut.edu.au Queensland Cyber Infrastructure Foundation (QCIF)

- Magdalena Antczak magdalena.antczak@qcif.edu.au

## Project description and aims

Aims of the project:
•	Design, develop, test, and optimise an end-to-end nf-ONTvisc workflow for the diagnosis of plant and insect viruses using ONT long read data collected via whole genome metagenomics sequencing and / or amplicon targeted sequencing.
•	Assess the performance of nf-ONTvisc workflow for running on diverse platforms including on-prem HPC and national compute infrastructure facilities (Pawsey Supercomputing Centre and the National Computational Infrastructure (NCI)).
•	Prepare a hands-on training user guide for the upskilling of end users and make the resource available through appropriate platforms.

The project's first phase was completed at the end of December 2023. Version 1.0 of the ONTViSc pipeline is now available on [GitHub](https://github.com/eresearchqut/ontvisc) and [WorkflowHub](https://workflowhub.eu/workflows/683). Detailed information can be found on [GitHub](https://github.com/eresearchqut/ontvisc). A step-by-step draft of a guide on how to set up and execute the ONTvisc pipeline on one of the HPC systems: Lyra (Queensland University of Technology), Setonix (Pawsey) and Gadi (National Computational Infrastructure) can be found [here](https://mantczakaus.github.io/ontvisc_guide/). 

## How is ABLeS supporting this work?

This work is supported through the software accelerator scheme provided by ABLeS. The support includes 0.5 TB long term storage, 1 TB temoprary storage on scratch and 10 KSUs per quarter.

## Expected outputs enabled by participation in ABLeS

ABLeS enables the ONTViSc pipeline to be tested using the Seqera platform on Gadi (NCI) and Setonix (Pawsey). This ensures that the workflow is compatible with the Seqera platform, Slurm and PBS-based computing environments and, hence, accessible to a wider community. 

> *These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above.*
