---
title: Nathan Lo, University of Sydney
description: The proposed project will investigate the genomic architecture, evolutionary history and population structure of multiple Blattodean species. This has substantial implications for our understanding of chromosome evolution, inbreeding and parallel evolution, as well as useful results for the conservation of insect biodiversity in Australia and Japan.
toc: false
type: ABLeS Participant
---

## Project title

Testing links between life-history and genome evolution

## Collaborators and funding

-   [The University of Sydney](https://meep.sydney.edu.au/)
-   [The University of Melbourne](https://www.wehi.edu.au/researcher/aaron-jex/)  
-   Australian Research Council Discovery Project DP240102805; [http://www.arc.gov.au/](http://www.arc.gov.au/)

## Contact(s)

-   Maxim Adams, <maxim.adams@sydney.edu.au>
-   Nathan Lo, <nathan.lo@sydney.edu.au>

## Project description and aims

Our aims are to:
-   Assemble high-quality genomes from species across the subfamily Panesthiinae with which to investigate the presence of genome-wide parallel evolution associated with soil-burrowing behaviour. If detected, this would represent among the first evidence that complex behavioural traits can emerge via parallel molecular trajectories.
-   Assemble a high-quality reference genome for the Japanese termite Glyptotermes nakajimai against which to align and call genome-wide SNPs in samples across the species’ range. These data will then be used to investigate population structure, inbreeding and genetic diversity in different colonies and populations. When combined with karyotype data, we will novelly test for an association between inbreeding and neo-sex chromosome formation.
-   Perform maximum-likelihood and Bayesian phylogenetic analyses to contextualise the previous results against the lineages’ evolutionary histories. These methods will also shed light into the biogeographic origins of the species.
 
We request >= 128 GB of RAM (due to the computational demands of these bioinformatic pipelines) and >= 2 TB of storage (due to the large size of raw genomic data). Requested dependencies include: GATK, PLINK, StringTie, Flye, bedtools, samtools, MrBayes, BEAST, IQ-TREE and STRUCTURE.

## How is ABLeS supporting this work?

Infrastructure choice: Pawsey
Quarterly Service Units (estimate): 100 kSUs
Storage: >= 2 Tb
Is access to GPUs required?: Not initially, but we may need them in due course (in a month or so, but it will depend on the approach our new computational specialist wants to use for analyses). 
Expected number of users: 4
Expected duration of the project: 3 years

## Expected outputs enabled by participation in ABLeS

Publications in scientific journals such as Current Biology, Molecular Biology and Evolution, PNAS, Evolution.

<br/>

> _These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above._
