---
title: Population Health and Immunity, WEHI.
description: This project aims to establish a robust metagenome assembled genome (MAG) database specifically focussing on bacterial communities from freshwater and wastewater ecosystems. Development of comprehensive metagenome assembled genome (MAG) database can provide essential insights into these ecosystems, ultimately supporting/allowing researchers to easily access and utilize the genomic data for their studies and decision-making.
toc: false
type: ABLeS Participant
---

## Project title

Development of metagenome assembled genomes (MAGs) of bacteria from freshwater and wastewater ecosystem.

## Collaborators and funding

- [WEHI](https://www.wehi.edu.au)

- [Melbourne Water](https://www.melbournewater.com.au)

## Contact(s)

- Aaron R Jex <jex.a@wehi.edu.au>

- Nijoy John <john.n@wehi.edu.au>

- Leon Hartman <hartman.l@wehi.edu.au>

- Imadh Azeez <azeez.i@wehi.edu.au>

- Balu Balan <balan.b@wehi.edu.au>

## Project description and aims

Lack of reference genomes for bacterial communities in freshwater and wastewater ecosystems leading to gaps in our understanding of microbial diversity and functional potential of bacteria in these ecosystems.

This project aims to develop a comprehensive metagenome assembled genome (MAG) database focused on cyanobacteria and other bacterial communities in freshwater and wastewater ecosystems. A major challenge in understanding these environments is the lack of reference genomes for their diverse bacterial communities. This gap significantly limits our ability to accurately identify microbial taxa, elucidate their functional roles, and assess ecological dynamics. By addressing the challenges posed by the lack of reference genomes, we aim to enhance our ability to monitor, manage, and conserve these vital ecosystems.

## How is ABLeS supporting this work?

This work is supported through the production bioinformatics scheme provided by ABLeS. The supports includes unlimited temporary storage on scratch, 15 TB long term storage and 50 KSUs per quarter.

## Expected outputs enabled by participation in ABLeS

We expect that a user-friendly database interface will be created to ensure accessibility for researchers, policymakers, and environmental managers, promoting the utilization of this valuable resource in research, monitoring, and biotechnological applications. Additionally, the incorporation of novel MAGs into existing metagenomic analysis pipelines will enrich the current databases and enhance the capacity for research in microbial ecology.

We look forward to publishing the results in high impact journals as well as the MAGs generated from these studies will be deposited in public databases (for e.g., NCBI database). Pipelines used/developed to generate MAGs will be accessible from Github page.

<br/>

> _These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above._
