---
title: Transcriptomics team at ZERO Childhood Cancer
description: Analysis of high-risk paediatric children enrolled in the Zero Childhood Cancer initiative.
toc: false
type: ABLeS Participant
---

## Project title
Transcriptomics analysis for ZERO Childhood Cancer

## Collaborators and funding

- Minderoo Foundation
- The Medical Research Future Fund
- The Australian Cancer Research Foundation
- Luminesce Alliance – Innovation for Children’s Health

## Contact(s)

- Chelsea Mayoh <CMayoh@ccia.org.au>


## Project description and aims

The aim of the project is to develop and analyse the splicing profiles and structural rearrangement events within the RNA samples that cannot be identified through fusion detection algorithms. This project will develop the pipeline and then analyse all 700+ patients to identify novel structural rearrangement events occurring in the RNA and unique splicing profiles that are specific to high-risk paediatric cancers. The development of this pipeline and analysis will then be integrated into the current RNA pipeline that will be ran prospectively on all new patients enrolled on ZERO.

The analysis will be carried out using [MINTIE](https://github.com/Oshlack/MINTIE) and rMATS. Both methodologies will be tested and compared to determine which algorithm to continue using, or if both are required.


## How is ABLeS supporting this work?

This work is supported through the production bioinformatics scheme provided by ABLeS. The supports includes 1 TB temprary storage, 5 TB long term storage and 100 KSUs per quarter.

## Expected outputs enabled by participation in ABLeS

A computational workflow that identifies cryptic structural rearrangements and alternate splicing events from bulk RNA-sequencing data that will be fully integrated with whole genome sequencing data.

We will analyse >1000 high-risk paediatric cancer patients enrolled on the ZERO childhood cancer precision medicine program and fully characterise cryptic structural rearrangements and alternate splicing events from RNA sequencing patient to be used in a clinically relevant time frame to identify drives of the patient’s disease. In addition, we will also expand this workflow to characterise the splicing landscape and structural rearrangements in all children with cancer in Australia to identify potential novel drivers of disease and alterations with targetable therapy options.

The data generated from this project will be made publicly available, with workflows published on Github.

<br/>

> *These details have been provided by project members at project initiation. For more information on the project, please consult the contact(s) or project links above.*
